distributed_shampoo/tests/distributed_shampoo_test.py

```diff
@@ -149,18 +148,6 @@ def test_invalid_with_incorrect_hyperparameter_setting(self) -> None:
                     **incorrect_hyperparameter_setting,
                 )
 
-    def test_invalid_cuda_pytorch_compile_setting(self) -> None:
```

Comment:
See my earlier question on the removal of the corresponding code in `DistributedShampoo`.